happybase/pool.py

```diff
@@ -48,10 +48,14 @@ class ConnectionPool(object):
     task of the pool.
 
     :param int size: the maximum number of concurrently open connections
+    :param bool autoconnect: Whether a connection should be created upon
+        instantiation to test connection availability.
+        Note: the `autoconnect` flag of Connection instance inside the pool
+        is always False.
```

Comment:
Can you rephrase the second part as "Note that this is unrelated to the `autoconnect` argument for connections, which is not applicable to connections managed by a connection pool." ? thanks